### PR TITLE
Support content attachments api

### PR DESCRIPTION
**Pull request by nadavkaner**

### CODE REVIEW

**Comment by nadavkaner** [comment]
@gmlewis Done 👍🏼

File: github/apps.go [hunk 1]
```diff
@@ -183,6 +190,28 @@ func (s *AppsService) CreateInstallationToken(ctx context.Context, id int64) (*I
...
+
+	// TODO: remove custom Accept headers when APIs fully launch.
+	req.Header.Set("Accept", mediaTypeReactionsPreview)
+
+	m := &Attachment{}
+	resp, err := s.client.Do(ctx, req, m)
+	if err != nil {
+		return nil, resp, err
+	}
+	return m, resp, nil
```
Comment (gmlewis) [inline github/apps.go]: Please add a blank line before the return statement.

Comment (nadavkaner) [reply github/apps.go]: @gmlewis 👍

**Comment by gmlewis** [comment]
Thank you, @gauntface!
Merging.

**Comment by gmlewis** [comment]
Fixes #1073.

**Pull request merged**